bin/merged_prs.py

```diff
@@ -7,29 +7,24 @@
 gh = Github()
 
 # Needed data
-first_tag = "v0.9.0"
-last_tag = "v0.9.1"
-
-# Running
-org = gh.get_organization('scalameta')
-repo = org.get_repo('metals')
+first_tag = "v0.9.1"
+last_tag = "v0.9.2"
 
 tag_range = "%s..%s" % (first_tag, last_tag)
-command = ['git', 'log', tag_range, "--first-parent", "main", "--pretty=format:\"%s\""]
+command = ['git', 'log', tag_range, "--first-parent", "main", "--pretty=format:%H"]
 process = subprocess.Popen(command, stdout=PIPE, stderr=PIPE)
 stdoutput, stderroutput = process.communicate()
 
-all_prs = []
-for line in stdoutput.split("\n"):
-    pr_num = re.findall("#\d+", line)
-    if len(pr_num) > 0:
-      all_prs.append(int(pr_num[0][1:]))
-
-for pr in all_prs:
-    try:
-        pull = repo.get_pull(pr)
-        print ("- %s" % pull.title)
-        print ("[\#%s](%s)" % (pull.number, pull.html_url))
-        print ("([%s](https://github.com/%s))" % (pull.user.login, pull.user.login))
-    except Exception:
-        print ("Cannot read PR %s" % pr)
+found_prs = []
+for sha in stdoutput.split("\n"):
+    main = stdoutput[0]
+    query = "repo:scalameta/metals type:pr SHA %s" % sha
+    for issue in gh.search_issues(query):
```

Comment:
This now searches for an exact PR that the commit was added in. It should help in case the PR didn't produce a merge commit.